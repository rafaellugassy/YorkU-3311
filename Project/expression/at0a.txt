union

start_of_set_enumeration
bool_value(true)
bool_value(true)
bool_value(true)
end_of_set_enumeration

start_of_set_enumeration
bool_value(false)
bool_value(true)
bool_value(true)
simplify
end_of_set_enumeration

simplify
