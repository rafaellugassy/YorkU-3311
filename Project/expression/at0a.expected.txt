  Expression currently specified: ?
  Report: Expression is initialized.
->union
  Expression currently specified: (? \/ nil)
  Report: OK.
->start_of_set_enumeration
  Expression currently specified: ({?} \/ nil)
  Report: OK.
->bool_value(True)
  Expression currently specified: ({True, ?} \/ nil)
  Report: OK.
->bool_value(True)
  Expression currently specified: ({True, True, ?} \/ nil)
  Report: OK.
->bool_value(True)
  Expression currently specified: ({True, True, True, ?} \/ nil)
  Report: OK.
->end_of_set_enumeration
  Expression currently specified: ({True, True, True} \/ ?)
  Report: OK.
->start_of_set_enumeration
  Expression currently specified: ({True, True, True} \/ {?})
  Report: OK.
->bool_value(False)
  Expression currently specified: ({True, True, True} \/ {False, ?})
  Report: OK.
->bool_value(True)
  Expression currently specified: ({True, True, True} \/ {False, True, ?})
  Report: OK.
->bool_value(True)
  Expression currently specified: ({True, True, True} \/ {False, True, True, ?})
  Report: OK.
->simplify
  Expression currently specified: ({True, True, True} \/ {False, True, True, ?})
  Report: Error (Expression is not yet fully specified).
->end_of_set_enumeration
  Expression currently specified: ({True, True, True} \/ {False, True, True})
  Report: OK.
->simplify
  Expression currently specified: ({True, True, True} \/ {False, True, True})
  Report: {True, False}
